src/gt4py/next/program_processors/codegens/gtfn/gtfn_module.py

```diff
@@ -135,8 +135,9 @@ def _process_connectivity_args(
                 # connectivity argument expression
                 nbtbl = (
```

Comment:
```suggestion
                assert len(connectivity_type.domain) == 2
                nbtbl = (
```

Reply:
this is already checked because it's a `NeighborConnectivityType`, but I could use `source_dim` and `neighbor_dim` instead of by index to make it more explicit.